test_scripts/Defects/7_2/2380_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua

```diff
@@ -0,0 +1,71 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2380
+--
+-- Description:
+-- PoliciesManager must allow all requested <vd_param> parameters in case "parameters" field is omitted
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) SubscribeVehicleData, UnsubscribeVehicleData, OnVehicleData RPCs are allowed by policies
+-- 3) "parameters" field is omitted at PolicyTable for all RPCs
+-- 4) App is registered
+--
+-- In case:
+-- 1) App sends valid SubscribeVehicleData(<vd_param>=true) request to SDL
+-- SDL does:
+-- - a) transfer this request to HMI
+-- 2) HMI sends SubscribeVehicleData response with <vd_param> data to SDL
+-- SDL does:
+-- - a) send SubscribeVehicleData response with (success = true, resultCode = "SUCCESS",
+--   <vd_param> = <data received from HMI>) to App
+-- 3) HMI sends valid OnVehicleData notification with <vd_param> parameter data to SDL
+-- SDL does:
+-- - a) transfer this notification to App
+-- Exception: Notification for unsubscribable VD parameter is not transfered
+-- 4) App sends valid UnsubscribeVehicleData(<vd_param>=true) request to SDL
+-- SDL does:
+-- - a) transfer this request to HMI
+-- 5) HMI sends VI.UnsubscribeVehicleData response with <vd_param> parameter to SDL
+-- SDL does:
+-- - a) send UnsubscribeVehicleData response with (success = true, resultCode = "SUCCESS",
+--    <vd_param> = <data received from HMI>) to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/common')
+
+--[[ Local Function ]]
+local function pTUpdateFunc(pt)
+  local VDgroup = {
+    rpcs = {
+      [common.rpc.sub] = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+      },
+      [common.rpc.on] = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+      },
+      [common.rpc.unsub] = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+      }
```

Comment:
I would explicitly define for each RPC that `parameters` field is omitted. E.g.:
```
      [common.rpc.sub] = {
            parameters = nil,
            hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
}


```